packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -472,17 +478,23 @@ class _AnimatedFadeOutFadeInState extends ImplicitlyAnimatedWidgetState<_Animate
 
   @override
   Widget build(BuildContext context) {
+    final Widget target = FadeTransition(
+      opacity: _targetOpacityAnimation,
+      child: widget.target,
+    );
+
```

Comment:
Nit: changes from 481-485 seems to be unnecessary. Adding lines 486-489 is enough?

Reply:
I'm reusing the target variable declared on 481.  Where should I move it?

Reply:
Oh, I though you were returning `widget.target` instead of `target`. Since the `FadeTransition` already completes, why can't we directly return `widget.target`?

Reply:
Never mind, I now see that `FadeTransition` is using `_targetOpacityAnimation`, not `_placeholderOpacityAnimation`. Ignore this nit :)